aria/cli/args_parser.py

```diff
@@ -137,10 +137,6 @@ def add_workflow_parser(workflow):
         '-w', '--workflow',
         default='install',
         help='The workflow name')
-    workflow.add_argument(
```

Comment:
putting aside the fact this code is going to be thrown out anyway,
how would a workflow execution not require a service instance to run on?